src/test/java/org/broadinstitute/hellbender/engine/dataflow/coders/GATKReadCoderUnitTest.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.hellbender.engine.dataflow.coders;
+
+import com.google.cloud.dataflow.sdk.Pipeline;
+import com.google.cloud.dataflow.sdk.testing.DataflowAssert;
+import com.google.cloud.dataflow.sdk.testing.TestPipeline;
+import com.google.cloud.dataflow.sdk.transforms.Create;
+import com.google.cloud.dataflow.sdk.values.PCollection;
+import org.broadinstitute.hellbender.utils.dataflow.DataflowUtils;
+import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+
+public class GATKReadCoderUnitTest extends BaseTest {
+
+    private GATKRead makeGoogleRead( final int uuid, final String name, final int start, final int length ) {
+        return ArtificialReadUtils.createGoogleBackedReadWithUUID(new UUID(0, uuid), name, start, length);
+    }
+
+    public GATKRead makeSamRead( final int uuid, final String name, final int start, final int length ) {
```

Comment:
public?
Also, FYI, these will get more to a common util after the read pre-processing pipeline is ready.